projects/transformers/finetuning_constants.py

```diff
@@ -57,6 +59,7 @@
     "qnli": ["accuracy"],
     "qqp": ["accuracy", "f1"],
     "rte": ["accuracy"],
+    "squad": ["exact_match", "f1"],
```

Comment:
What sort of metric is exact match, is that a specific way of measuring performance?